terraform/ec2/mac/main.tf

```diff
@@ -135,32 +135,6 @@ resource "null_resource" "integration_test" {
   }
 
 }
-
```

Comment:
This looks like an accidental bad merge. We don't need this since the previous `resource` does the actual integration test.